exotic.py

```diff
@@ -25,7 +25,7 @@
 # Major releases are the first digit
 # The next two digits are minor commits
 # (If your commit will be #50, then you would type in 0.5.0; next commit would be 0.5.1)
-versionid = "0.7.5"
```

Comment:
Let's not increment the versionid here - let's do that when we merge dev into main. Also, I like to incrementally update, so the next versionid would be 0.7.6. Please remove your edit on this line for now.

Reply:
Given the scope of the changes, the [Semantic Versioning Specification](https://semver.org/) implies that the [minor version](https://semver.org/#spec-item-7) (e.g. **Major.Minor.Patch**) should be updated given that the API ingestion change is a fairly different and new piece of functionality. But this is not a steadfast rule, of course, just a suggestion.

Reply:
I'll let whoever merges with main to handle this